kubernetes/samples/scripts/create-rcu-schema/README.md

```diff
@@ -233,13 +202,3 @@ Pod [rcu] removed from nameSpace [default]
 Use this script to stop the Oracle DB service you created using the `start-db-service.sh` script.
```

Comment:
Script name to stop the Database service is missing.

Reply:
Added the line 

Stop an Oracle Database service in a Kubernetes cluster
Use the script samples/scripts/create-oracle-db-service/stop-db-service.sh